core.php

```diff
@@ -504,23 +504,23 @@
-    "update_available"          => "Update Available",
+    "results_per_page"          => "Resultaten per pagina",
+    "class_name"                => "Klasse Naam",
+    "possible_permission_issue" => "U voert de opdracht uit als  \":cli_user\", terwijl \":path\" is eigendom van \":path_user\", dit kan leiden tot problemen met bestandsrechten.",
+    "system_update"             => "Systeem Update",
+    "system_update_desc"        => "Update SupportPal naar de nieuwste beschikbare versie.",
+    "update_available"          => "Update beschikbaar",
     "up_to_date"                => "Up to Date",
-    "release_notes"             => "Release Notes",
-    "about_to_update_to"        => "You are about to update to version <strong>:version</strong>. A full system backup will be taken automatically prior to the update.",
-    "waiting_for_update_start"  => "Waiting for the update to start... This may take a few minutes.",
-    "binary_paths"              => "Binary Paths",
-    "binary_paths_desc"         => "The PATH environment variable is used to find required binaries. If one or more binary cannot be found, you can specify the full path for each binary file.",
+    "release_notes"             => "Release Notities",
+    "about_to_update_to"        => "U staat op het punt bij te werken naar versie <strong>:version</strong>. Voorafgaand aan de update wordt automatisch een volledige systeemback-up gemaakt.",
+    "waiting_for_update_start"  => "Wachten tot de update start... Dit kan enkele minuten duren.",
+    "binary_paths"              => "Systeem Paden",
+    "binary_paths_desc"         => "De omgevingsvariabele PATH wordt gebruikt om vereiste binaire bestanden te vinden. Als een of meer binaire bestanden niet kunnen worden gevonden, kunt u het volledige pad voor elk binair bestand opgeven.",
     "cron_makesure"             => "Voor hulp met instellen van de cron job, zie <a target='_blank' href='https://docs.supportpal.com/current/Configuring+the+Cron'>Cron Job Help</a>.",
-    "cron_must_run_for_update"  => "The cron job must be running to use this function.",
-    "support_expired"           => "The support and updates on your license have expired.",
-    "update_permitted_version"  => "The license only permits updating to :version.",
-    "update_renew_support"      => "To update to the latest version of SupportPal, please renew your support and updates.",
-    "renew"                     => "Renew",
+    "cron_must_run_for_update"  => "De Cron Job moet actief zijn om deze functie te gebruiken .",
+    "support_expired"           => "De ondersteuning en updates van uw licentie zijn verlopen.",
+    "update_permitted_version"  => "De licentie staat alleen updaten toe naar :version.",
+    "update_renew_support"      => "Vernieuw uw ondersteuning en updates om bij te werken naar de nieuwste versie van SupportPal.",
+    "renew"                     => "Vernieuw",
```

Comment:
Don't want to nitpick but I think `Verleng` instead of `Vernieuw` has a better ring to it.